doc/JSON_INFO.md

```diff
@@ -3470,6 +3470,28 @@ Any Item can be a container. To add the ability to contain things to an item, yo
 "flags" : ["CHOP"],    // Indicates special effects
 "to_hit": 1            // To-hit bonus if using it as a melee weapon
 ```
+### Memory Cards
+
+Memory card information can be defined on any GENERIC item by adding an object named `memory_card`, this field does not support `extend`/`remove`, only override.
+
+```C++
+"id": "memory_card_unread",
+"name": "memory card (unread)",
+// ...
+"memory_card": {
+  "data_chance": 0.5,                  // 50% chance to contain data
+  "on_read_convert_to": "memory_card", // converted to this itype_id on read
+  "photos_chance": 0.33,               // 33% chance to contain new photos
+  "photos_amount": 3,                  // contains between 1 and 3 new photos
+  "songs_chance": 0.33,                // 33% chance to contain new songs
+  "songs_amount": 4,                   // contains between 1 and 4 new songs
+  "recipes_chance": 0.33,              // 33% chance to contain new recipes
+  "recipes_amount": 5,                 // contains between 1 and 5 new recipes
+  "recipes_level_min": 4,              // recipes will have at least level 4
+  "recipes_level_max": 8,              // recipes will have at most level 8
+  "recipes_categories": [ "CC_FOOD" ]  // recipes from CC_FOOD category
```

Comment:
Could there be a json object for `"SECRET": true` //recipes will come from secret recipes in this category?  That way we could have memory cards in laboratories with a chance to have a single mutagen recipe or something else hidden?

Reply:
do you mean filtering by recipe "flags"? that's also doable i suppose, if CC_CHEM and level>6 isn't a good enough filter

Reply:
I don't want to request scope creep if I don't have a planned use case that I'm going to implement immediately.

Reply:
Alright, give a poke if you need help implementing some specific filter, should be fairly easy now - add fields to the struct, read them from json and check the predicate in the activity